tests/actions/pastOrders.test.js

```diff
@@ -0,0 +1,41 @@
+import expect from 'expect';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import moxios from 'moxios';
+import { LOAD_PAST_ORDERS_SUCCESS, LOAD_PAST_ORDERS_FAILURE } from '../../app/js/actions/actionTypes';
+import mockData from '../../__mocks__/mockData';
+import locationMock from '../../__mocks__/locationMock';
+import { getPastOrders } from '../../app/js/actions/pastOrders';
+
+const middlewares = [thunk];
```

Comment:
some of these variables have been declared globally, so no need to declare them locally again, e.g. thunk, expect, etc please refer to tests/setup.js

Reply:
Worked on